### PR TITLE
corporate tax id maybe start with zero

### PR DESCRIPTION
**Pull request by iamsarin**
I fix replace mobile number start 0 to 66 but some corparate tax id maybe start with zero

### CODE REVIEW

File: browser.js [hunk 1]
```diff
@@ -0,0 +1,2 @@
+var generatePayload = require('./src/generatedPayload')
+window.generatePayload = generatePayload
```
Comment (dtinth) [inline browser.js]: This file is not needed (we can use UMD format)

File: package.json [hunk 2]
```diff
@@ -2,7 +2,9 @@
   "dependencies": {
     "crc": "^3.4.4",
     "minimist": "^1.2.0",
-    "qrcode": "^0.9.0"
+    "qrcode": "^0.9.0",
+    "unminified-webpack-plugin": "^1.2.0",
```
Comment (dtinth) [inline package.json]: I just knew about this. Thanks!

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,21 @@
+var path = require('path')
+var webpack = require('webpack')
+var UnminifiedWebpackPlugin = require('unminified-webpack-plugin')
+
+module.exports = {
+  entry: {
+    index: './browser.js'
```
Comment (dtinth) [inline webpack.config.js]: This can point directly to `generatePayload.js`.

File: webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+  },
+  output: {
+    path: path.resolve(__dirname, 'dist'),
+    filename: 'promptypay-browser.min.js'
```
Comment (dtinth) [inline webpack.config.js]: We can use `output.library` to generate a UMD library (works with CommonJs, AMD, and globals.)

For more details: https://webpack.js.org/configuration/output/#output-librarytarget

**Comment by iamsarin** [comment]
I sorry for my PR mistake. I intend PR only fix corporate tax id.

**Comment by dtinth** [comment]
I have cherry picked your changes and released v0.4.3.

742cd3486c7cefbfc71a3869b59cf72b5a1cedcd

**Comment by dtinth** [comment]
That’s a really good catch. Thanks for the contribution again. :)

**Pull request closed without merging**